### ISSUE TITLE
fix: do not compute the std when reading the data

### ISSUE THREAD

**Issue by iomz** [Interactions-HSG/MyoKTROS#22]
For the Keras model, the [preprocessing layer](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Normalization) can take care of the computation.
Instead, return a feature of shape `(N_SENSORS*n_samples,)` to increase the input size.

**Comment by iomz** [comment]
This is apparently not right. Computing the variances per n_samples and feeding them as inputs seem to have higher robustness. Need to update the data stream pipeline once again.

**Comment by iomz** [comment]
I am adding an option to switch the queueing structure
1. mean values and variances for n_samples: `shape == (8*2, )`
2. flattened n_samples: `shape == (8*n_samples, )`
